app/helpers/complaints_helper.rb

```diff
@@ -4,4 +4,9 @@ def confirm_btn_label(complaint)
     return "Potwierdź" if confirmers_count.zero?
     "Potwierdź (#{confirmers_count})"
   end
+
+  def confirm_btn_class(complaint, user)
+    return "btn btn-primary" unless user
+    complaint.confirmers.include?(user) ? "btn btn-primary" : "btn btn-outline-primary"
```

Comment:
`if user_signed_in?` in the view means that the user will be always passed to this method so I think there's no need for the early return here. I would rather check for the user not included in the confirmers and set the `outline` class otherwise.